### ISSUE TITLE
Data Issue: Hawaii case number discrepancy 

### ISSUE THREAD

**Issue by kw468** [nytimes/covid-19-data#453]
On the 9/12 update from Hawaii, 131 cases were added, bringing the total to 10,588 (source: https://health.hawaii.gov/coronavirusdisease2019/). However, NYT is showing 10,696. What is driving this difference? It is not due to probable cases, as the weekly change from last week was +3 (source: https://health.hawaii.gov/coronavirusdisease2019/what-you-should-know/current-situation-in-hawaii/). All other data points for last week match the Hawaii noon updates.

**Comment by mching** [comment]
I noticed this too. 

Here is the county breakdown ([HDOH](https://health.hawaii.gov/coronavirusdisease2019/), NYT): 
Hawaii 572, 579
Honolulu 9559, 9616
Kauai 58, 58
Maui 373, 443

**Comment by kw468** [comment]
Right, so difference between HDOH and NYT corresponds to the Total Probable Cases (134 in total) found at [health.hawaii.gov](https://health.hawaii.gov/coronavirusdisease2019/what-you-should-know/current-situation-in-hawaii/). The 9/8 [snapshot](https://web.archive.org/web/20200909055735/https://health.hawaii.gov/coronavirusdisease2019/what-you-should-know/current-situation-in-hawaii/) of that page shows 131 probable cases, hence the NYT 9/12 update should be +3 probable cases, not +134. I did not check if the 131 were already included in the 9/5 NYT update.

**Comment by mching** [comment]
Sorry, I haven't been comparing the DOH and NYT data or the probables until I noticed the big jump today. 

As far as I can tell the problem is not just in probable cases but also in the county counts. Here's the [comparison of the data](https://docs.google.com/spreadsheets/d/1k94QT4cwyqTiYBu-xaSHuBJWDytpbRILLjfTzM2jJLQ/edit?usp=sharing) I pulled from the NYT counties file with comparison to the [Hawaii data](https://docs.google.com/spreadsheets/d/1WTUg75_mm35Z2C6PQd5AOKtVeSNfci5Mp0G_pgYTqQI/edit#gid=0) that Ryan Ozawa (and maybe you?) have been curating. It's almost identical every day except for 9/12.

**Comment by lwaananenjones** [comment]
Your assumptions here are correct — the addition of probable cases in Hawaii is currently in our dataset for 9/12, which is the first day we were able to verify methodology with the state health department. We're planning to revise the counts back a few days to 9/8 when probable case data was first published. On our [tracker page](https://www.nytimes.com/interactive/2020/us/hawaii-coronavirus-cases.html), there is an anomaly note when those cases were added; the date for that note will be adjusted to 9/8 once we have the revision in place.
